config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root to: 'users#index'
+  get 'users/:id', to: 'users#show'
+  get 'users/:user_id/posts', to: 'posts#index'
+  get 'users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] [OPTIONAL] Kindly use the following way to define your routes as it's more efficient and easier to read.
 ```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

Reply:
@gemmen29 Thank you so much.